toolchest/yaml.py

```diff
@@ -4,6 +4,17 @@
 import yaml
 
 
+class IndentDumper(yaml.Dumper):
+    """ Custom yaml dumper to ensure elements are indented """
+    def increase_indent(self, flow=False, indentless=False):
+        # We can't set indentless from the yaml.dump call and is being set to
+        # True. We need it to be False.
+        state = indentless
+        if indentless:
+            state = False
+        return super().increase_indent(flow, state)
```

Comment:
Again, possibly being dense, but why not just set this explicitly to False in this call, if that is what we always want? I know you need to have indentless in the signature because you are subclassing, but it seems like setitng False directly in the implementation would simplify the code?

Reply:
yes it would and that is what I had before. Let me drop that but keep the comment.

Reply:
Agreed just a comment would be fine!